packages/expo-background-fetch/src/BackgroundFetch.types.ts

```diff
@@ -42,7 +42,7 @@ export interface BackgroundFetchOptions {
   /**
    * Inexact interval in seconds between subsequent repeats of the background fetch alarm. The final
    * interval may differ from the specified one to minimize wakeups and battery usage.
-   * - On Android it defaults to __15 minutes__,
+   * - On Android it defaults to __10 minutes__,
```

Comment:
To see the updated interval in docs you need to regenerate API data JSON file. To do that need to to run Expo Tools command:
* `et gdad -p expo-background-fetch`

Also package source rebuild could be needed to updated the exported types.